libs/blog/newsletter/feature-newsletter/src/lib/feature-newsletter/newsletter.component.html

```diff
@@ -38,16 +38,20 @@ <h2 class="text-4xl font-bold">{{ t('title') }}</h2>
                 class="text-al-muted flex flex-row peer-checked:[&>div>div]:block"
                 for="newsletter-checkbox"
               >
-                <div
-                  class="border-al-muted mr-2 mt-1 flex aspect-square h-[14px] items-center justify-center rounded border"
-                >
+                @if (newsletterStoreLoading() !== 'error') {
                   <div
-                    class="bg-al-pink hidden h-[6px] w-[6px] rounded-sm"
-                  ></div>
-                </div>
-                <span class="before:text-al-pink before:content-['*']">
-                  {{ t('consent') }}
-                </span>
+                    class="border-al-muted mr-2 mt-1 flex aspect-square h-[14px] items-center justify-center rounded border"
+                  >
+                    <div
+                      class="bg-al-pink hidden h-[6px] w-[6px] rounded-sm"
+                    ></div>
+                  </div>
+                  <span class="before:text-al-pink before:content-['*']">
+                    {{ t('consent') }}
+                  </span>
+                } @else {
+                  <p class="font-bold">{{ t('error') }}</p>
```

Comment:
I decided to use `font-bold` instead `text-al-primary` class because `text-al-primary ` does not provide good contrast